src/routes/routes.js

```diff
@@ -41,7 +41,7 @@ export const initRoutes = app => {
   app.get('/post', (req, res) => res.render('userPost', { isAuth: true }));
   app.get('/jobInvites', renderJobInvitesPage);
   app.get('/post/:inviteId', renderSinglePostPage);
-  app.get('/admin/reported', (req, res) => res.render('admin/reportedUsers', { isAuth: true, }));
```

Comment:
Why did you change this line to about?

Reply:
Oh, GitBash auto-merged that while I did an upstream pull I believe.
Route was already defined on that line. If you check my initial PR commit 
[a294cea](https://github.com/hngi/is-this-a-real-job/pull/79/commits/a294cea05e9ee967675e5f8d48bf2a6bf7d37462)

'/admin/reported', Route will be redefined on the line right after it in my next PR.